scripts/create_citation_items.rb

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "yaml"
+require "csv"
+require "fileutils"
+require "securerandom"
+require_relative "citation"
+
+# Creates citation register items from extracted citations
+class CitationItemCreator
+  attr_reader :citations_created, :start_identifier
+
+  def initialize(csv_file, registry_path, start_identifier)
+    @csv_file = csv_file
+    @registry_path = registry_path
+    @start_identifier = start_identifier
+    @citations_created = 0
+    @citation_id_map = {}
+    @citation_uuid_map = {}
+  end
+
+  def create_all
+    puts "📖 Reading citations from #{@csv_file}..."
+    
+    unless File.exist?(@csv_file)
+      raise "❌ Error: CSV file '#{@csv_file}' does not exist"
+    end
+
+    # Read CSV and parse citations
+    citations = read_citations_from_csv
+
+    puts "✅ Found #{citations.size} citations to create"
+    puts ""
+
+    # Create citation directory if it doesn't exist
+    citation_dir = File.join(@registry_path, "citation")
```

Comment:
This is not the correct thing, we do not simply create citations in the register.

Reply:
Yeah, I only focused on the extraction, since I wasn't sure how the "proposal" would work.

Can you elaborate the steps here?

Reply:
There exists a tested script that we used for importing extents. I don’t think a novel approach to this challenge is worth pursuing.